### PR TITLE
Added documentation about output modules and formatters #444

### PR DESCRIPTION
**Pull request by joachimmetz**
Added documentation about output modules and formatters #444

### CODE REVIEW

File: docs/sources/user/Output-format-l2tcsv.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# l2tcsv output format
+
+By default log2timeline.pl used the l2tcsv (or l2t_csv) output format. l2tcsv
```
Comment (rgayon) [inline docs/sources/user/Output-format-l2tcsv.md]: I would make it clearer that this is not the case anymore.

File: docs/sources/user/Releases-and-roadmap.md [hunk 2]
```diff
@@ -7,41 +7,40 @@ Plaso comes in 2 different forms of releases:
 
 **Note that the development release is for development, expect frequent updates and potential breakage.**
 
-If you do not plan to develop or live on the edge, regarding plaso, we highly recommend sticking with the packaged release. We will try to provide two packaged releases per year, a "summer" and "winter" release (depending on your location), but occasionally we will also do preview and release candidate packaged releases.
+If you do not plan to develop or live on the edge, regarding Plaso, we highly
+recommend sticking with the packaged release. We will try to provide a monthy
+release, if time permits and recommend using a Plaso no older than 6 months.
```
Comment (rgayon) [inline docs/sources/user/Releases-and-roadmap.md]: "a version of Plaso" maybe?

File: docs/sources/user/Output-format-l2tcsv.md (hunk 1 above)
Comment (joachimmetz) [reply docs/sources/user/Output-format-l2tcsv.md]: ack, added note.

File: docs/sources/user/Releases-and-roadmap.md (hunk 2 above)
Comment (joachimmetz) [reply docs/sources/user/Releases-and-roadmap.md]: done

**Comment by joachimmetz** [comment]
> Just a couple nits

thx, nits addressed

**Pull request merged**